sources/packages/web/src/components/common/Reports.vue

```diff
@@ -78,6 +80,17 @@ export default defineComponent({
             PROGRAM_YEAR_DROPDOWN_KEY,
           );
         }
+        if (event.changed.value === PROGRAM_AND_OFFERING_STATUS) {
```

Comment:
This condition does not need to be added since we have the check `institutionName._visible`.
Removing the condition would allow the form logic to be changed and this method would not need to be updated, following the same logic for the program year.